UnitedIncome/components
# [APP-3023] Navbar Component

### Description
> Please provide a brief description of your changes below.

This PR includes the global `Navbar` component. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Load it up in Storybook.
* There's multiple customization options.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-3023)
